XM_Dinner/105_Marowak.yaml

```diff
@@ -0,0 +1,22 @@
+species: Marowak
+ingamename: Bone Curry
+displayname: Bone Curry
+setname: Dinner
+tags: [Dinner]
+ball: [Cherish]
+item: Thick Club

```

Comment:
Probably should be a rare bone for thematic reasons, but this means balance changes (bonemerang over bone club and flare blitz over fire fang). Might be a bit strong even with those changes though unless dinner is noticeably stronger than standard/